tests/install_vm.py

```diff
@@ -19,13 +19,6 @@ def parse_args():
         default="kickstarts/test_suite.cfg",
         help="Path to a kickstart file for installation of a VM."
     )
-    parser.add_argument(
-        "--distro",
```

Comment:
I prefer keeping this option, because it's an user friendly feature, the purpose of it is that users don't have to search for URLs of Fedora or CentOS the internet. That's convenient for them. You can use `--url` anytime to override the default hard coded URL if you want.

I suggest removing only `choices` keyword to not limit the choices of distributions and removing the `data.variant` variable, but keeping the `--distro`. Possibly, you can extend or update the hard coded URls.

Reply:
Ok, my problem with it was that it was directly related to `--variant` for `virt-install`; having it solely as an URL convenience (without the `virt-install` snippet) makes sense. Thanks!